docs/_articles/en/getting-started/install.md

```diff
@@ -11,24 +11,149 @@ Install Visual Studio Code on any computer running macOS, Windows, or Linux. VS
 
 To install Visual Studio Code visit <https://code.visualstudio.com> and click the big green **Download** button. After the download finishes, open the installer and follow the steps to complete the installation.
 
-## Java Platform, Standard Edition Development Kit
+## Salesforce Extensions for Visual Studio Code
 
-Some features in Salesforce Extensions for VS Code depend upon the Java Platform, Standard Edition Development Kit (JDK). You need to have either version 8 or version 11 of the JDK installed.
+After you’ve installed Salesforce Extensions for VS Code’s dependencies, install the Salesforce Extension Pack extension from the Visual Studio Marketplace. The extension pack contains everything you need to develop Salesforce apps using Visual Studio Code.
 
-If you don’t already have version 11 or 8 of the JDK installed, see the [instructions on how to install and configure Java](./en/getting-started/java-setup).
+To install the extensions, visit <https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode> and click the green **Install** button. The installation process prompts you to open Visual Studio Code. After Visual Studio Code opens, a dialog box opens in the lower right corner, asking you to install the extension. Click **Install** to proceed.
```

Comment:
@ekapner - For the Java instructions, could we have it on a separate page without having that page on the sidebar menu?  So, have it as a blurb on this page like it is today with a link to all the java instructions.  The Java detail takes up 10x the space of the rest of the topics.  what do you think?

Reply:
@smaddox-sf - yep, the topic already existed, so now I just cross ref to it - but will leave it off the sidebar/toc.